### DISCUSSION TITLE
curl: (22) The requested URL returned error: 400

### DISCUSSION THREAD

**Discussion by klazustrem** [xvzc/SpoofDPI#122] [Q&A]
я не пойму как эту проблему решить и что нужно сделать, сижу на Sonoma 14.5

**Comment by PoorMouse** [comment]
Проверьте правильность запрашиваемого адреса. Код 400 означает, чтобы был сделан "плохой", поврежденный запрос.

А вообще не мучайте себя curl. Просто скачайте готовый бинарник со [страницы релизов](https://github.com/xvzc/SpoofDPI/releases) (под свою ОС и архитектуру), положите в любое место и добавьте этот путь в PATH.
